helios-integrations/helios-retrofit/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+    compile project(":helios-core")
+    compile project(":helios-annotations")
+    compile project(":helios-parser")
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+
+    kapt project(':helios-meta')
+    
+    compile 'com.squareup.retrofit2:retrofit:2.6.0'
+
+    testImplementation("com.squareup.okhttp3:mockwebserver:3.14.2")
```

Comment:
place those versions here https://github.com/47deg/helios/blob/master/build.gradle#L41 along the others